### ISSUE TITLE
[Catalog] Menu button is missing an a11y label

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#3651]
Repro steps:

- Enable voice over.
- Navigate to any component landing page.
- Select the menu button.

Expected behavior:
- A sensible label is read aloud for the menu button.

Actual behavior:
- The button's image name is read aloud.

---
https://www.pivotaltracker.com/story/show/157261495
